extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -402,6 +399,26 @@ export class AmpA4A extends AMP.BaseElement {
   }
 
   /**
+   * Returns preconnect urls for A4A. Ad network should overwrite in their
+   * Fast Fetch implementation and return an array of urls for the runtime to
+   * preconnect to.
+   * @return {!Array<string>}
+   */
+  getPreconnectUrls() {
+    return [];
+  }
+
+  /**
+   * Returns preconnect urls for A4A. Ad network should overwrite in their
+   * Fast Fetch implementation and return an array of urls for the runtime to
+   * preconnect to.
+   * @return {!Array<string>|string}
```

Comment:
So now one can be string while there other cannot?  Let's be consistent (personally I would be fine with just an array every time)